### PR TITLE
Fix wrong path in favicons

### PR DESCRIPTION
**Pull request by RafaelGoulartB**
Hi!
The favicons link gives a 404.
Should It have the path `/favicons/favicon-**x**.png` instead of only `/favicon-**x**.png`?

That's my first Pull Request ever so if I did something wrong please let me know.
Thank you :D

### CODE REVIEW

File: pages/_document.tsx [hunk 1]
```diff
@@ -22,19 +22,19 @@ class MyDocument extends Document {
           <meta name="msapplication-wide310x150logo" content="mstile-310x150.png" />
           <meta name="msapplication-square310x310logo" content="mstile-310x310.png" />
```
Comment (LauraBeatris) [inline pages/_document.tsx:23]: We should do the same for android related icons

**Comment by LauraBeatris** [comment]
Hey @RafaelGoulartB 

Thanks! Can you also fix the  path for android related favicons?

**Review by LauraBeatris — CHANGES_REQUESTED** [review]

**Comment by RafaelGoulartB** [comment]
@LauraBeatris 

Did you use some tool to make all this favicons in different formats?

I'm using this website https://favicon.io/favicon-converter/, but it don't give me all this favicons.

**Comment by LauraBeatris** [comment]
@RafaelGoulartB I use https://favicomatic.com/ a lot, it also transforms all the favicons to have a transparent background

**Comment by LauraBeatris** [comment]
@RafaelGoulartB Thanks! Merged 😄

**Pull request merged**